common/common.h

```diff
@@ -80,8 +80,8 @@ struct gpt_params {
     int32_t yarn_orig_ctx         = 0;     // YaRN original context length
     float   defrag_thold          = -1.0f; // KV cache defragmentation threshold
 
-    ggml_backend_sched_eval_callback cb_eval;
-    void * cb_eval_user_data;
+    ggml_backend_sched_eval_callback cb_eval = nullptr;
```

Comment:
@ggerganov @ngxson probably I am not a good C developer enough, but that's why I do love tests:
https://github.com/ggerganov/llama.cpp/actions/runs/8636661014 :+1: